tests/test_ResolveTest.py

```diff
@@ -241,6 +241,14 @@ def test_id_resolution(self):
             (1, "Ala. Code § 92"),
             (1, "Id. at 2000"),
         )
+        # Test resolving an Id. citation with a pin cite when the previous
+        # citation only has a placeholder page. We expect this to still resolve
+        # because it's possible to pin cite to an opinion with pending final
```

Comment:
Really? You can pin site using a page before you have a citation with the page? That's truly weird.

Reply:
Hmm, maybe not actually, I thought that people would just pin cite to whatever the document page number is (starting at 1) if the real page number was not available, but now I'm not sure. Is it better to expect this test to not resolve the Id. citation?

Reply:
Fixed in 6048422